### PR TITLE
Keeper updates

### PR DESCRIPTION
**Pull request by prashantkrv2020**

### CODE REVIEW

File: x/cdp/client/cli/query.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func GetQueryCmd() *cobra.Command {
 
 func QueryCdp() *cobra.Command {
 	return &cobra.Command{
-		Use:   "cdp [owner-addr]",
+		Use:   "cdp [owner-addr] [collateralType]",
 		Short: "cdp's information",
 		Args:  cobra.ExactArgs(1),
```
Comment (dheerajkd30) [inline x/cdp/client/cli/query.go]: ExactArgs(2) ?

File: x/cdp/keeper/params.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
+	k.paramSpace.GetParamSet(ctx, &params)
+	return params
+}
+
+func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
+	k.paramSpace.SetParamSet(ctx, &params)
+}
+
+func (k Keeper) getLiquidationRatio(ctx sdk.Context, collateralType string) sdk.Dec {
```
Comment (dheerajkd30) [inline x/cdp/keeper/params.go]: func name correction >> Get..

File: x/cdp/types/errors.go [hunk 3]
```diff
@@ -7,9 +7,21 @@ import (
...
-	ErrorInvalidFrom     = sdkerrors.Register(ModuleName, 102, "invalid from")
-	ErrorInvalidReceiver = sdkerrors.Register(ModuleName, 103, "invalid receiver")
-	ErrorInvalidCoins      = sdkerrors.Register(ModuleName, 104, "invalid coins")
-	ErrorInvalidAmount   = sdkerrors.Register(ModuleName, 105, "invalid amount")
+	ErrorInvalidField         = sdkerrors.Register(ModuleName, 101, "invalid field")
+	ErrorInvalidFrom          = sdkerrors.Register(ModuleName, 102, "invalid from")
+	ErrorInvalidReceiver      = sdkerrors.Register(ModuleName, 103, "invalid receiver")
+	ErrorInvalidCoins         = sdkerrors.Register(ModuleName, 104, "invalid coins")
+	ErrorInvalidAmount        = sdkerrors.Register(ModuleName, 105, "invalid amount")
+	ErrSample                 = sdkerrors.Register(ModuleName, 106, "sample error")
```
Comment (dheerajkd30) [inline x/cdp/types/errors.go]: let's follow one convention Err or Error prefix. update either one.

**Review by dheerajkd30 — CHANGES_REQUESTED** [review]

File: x/cdp/keeper/cdp.go [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+	cdpIDs.Ids = cdpIDList
+	store.Set(ownerAddrs, k.cdc.MustMarshalBinaryBare(&cdpIDs))
+}
+
+func (k Keeper) SetNextCdpId(ctx sdk.Context, id uint64) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.CdpIdKey, types.GetCdpIDBytes(id))
+}
+
+func (k Keeper) GetCdpIdsByOnwer(ctx sdk.Context, owner sdk.AccAddress) (types.CdpIdList, bool) {
```
Comment (dheerajkd30) [inline x/cdp/keeper/cdp.go]: Owner** spelling mistake

**Review by dheerajkd30 — CHANGES_REQUESTED** [review]

File: x/cdp/keeper/cdp.go (hunk 4 above)
Comment (dheerajkd30) [reply x/cdp/keeper/cdp.go]: udated.

**Review by dheerajkd30 — APPROVED** [review]

**Review by bsrinivas8687 — APPROVED** [review]

**Pull request merged**